GNUmakefile

```diff
@@ -25,7 +25,7 @@ bin/terraform:
 testacc: fmtcheck bin/terraform
 	PATH="$(CURDIR)/bin:${PATH}" TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout=60s
 
-acceptance: testversion5.6 testversion5.7 testversion8.0 testpercona5.7 testpercona8.0 testmariadb10.3 testmariadb10.8 testmariadb10.10 testtidb6.1.0
+acceptance: testversion5.6 testversion5.7 testversion8.0 testpercona5.7 testpercona8.0 testmariadb10.3 testmariadb10.8 testmariadb10.10 testtidb6.1.0 testrdsdb5.7 testrdsdb8.0
```

Comment:
I'm not completely convinced these tests can run as a part of all tests - that would need some AWS account to use in CI. And also some way to set endpoints automatically.

Because without your change, a person can run "make acceptance" without any settings whatsoever and it just runs all the tests.

Reply:
I will remove from the default run. For now one can just run RDS acct tests separately.